includes/class-woocommerce.php

```diff
@@ -254,7 +254,7 @@ private function is_request( $type ) {
 	 * @param  string $theme Theme slug to check.
 	 * @return bool
 	 */
-	private function is_active_theme( $theme ) {
```

Comment:
Since this is private now, my pref would be to move it out of the main class as a new core function and call that instead. Thoughts on that vs public method?